docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,202 @@
+title: Getting started with Dagster Cloud | Dagster Cloud
 
-noindex: True
+platform_type: "cloud"
 ---
 
-# Getting started with Dagster Cloud
+# Getting Started with Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Welcome to Dagster Cloud!
+
+In this guide, we'll give you everything you need to get up and running with Dagster Cloud. We'll walk you through setting up your Dagster Cloud account, selecting a deployment type, loading your code, launching your first run, and configuring continuous integration.
+
+---
+
+## Prerequisites
+
+To complete the steps in this guide, you'll need access to some Dagster code.
+
+You can either use your own code or our [sample code (`dagster-cloud-examples`)](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). This guide uses the `dagster-cloud-examples` Python package.
+
+---
+
+## Step 1: Create a Dagster Cloud account and organization
+
+If you don't already have a Dagster Cloud account, [sign up for one](https://dagster.cloud/) before continuing.
+
+You'll also be asked to create a **Dagster Cloud organization.** When your organization is created, a single instance deployment, named `prod`, will also be created. Instance deployments are similar to environments in that they allow you to operate independent instances of Dagster, with separately managed permissions.
+
+---
+
+## Step 2: Choose your deployment type
+
+In this step, you'll choose how your code is hosted and executed:
+
+- [Hybrid deployment](#hybrid-deployment)
+- [Serverless deployment](#serverless-deployment)
+
+### Hybrid deployment
+
+With a hybrid deployment, your Dagster code is executed in your infrastructure.
+
+Using a hybrid deployment requires setting up an agent for use with Dagster Cloud. The agent runs in your infrastructure, executes your code, and streams metadata about code execution over HTTPS back to Dagster Cloud.
+
+Dagster Cloud currently supports the following agents:
+
+- [Amazon Elastic Container Service (ECS)](/dagster-cloud/deployment/agents/amazon-ecs/creating-ecs-agent-new-vpc)
+- [Local](/dagster-cloud/deployment/agents/local)
+- [Docker](/dagster-cloud/deployment/agents/docker)
+- [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
+
+**To use this deployment type**, follow the steps for setting up and deploying your agent of choice. Continue to the next step when finished.
+
+### Serverless deployment
+
+<Note>
+  Serverless deployment is currently in <strong>early access</strong>.
+</Note>
+
+Serverless deployment is a fully-managed version of Dagster Cloud that allows you to run your Dagster jobs without spinning up any infrastructure. While Serverless is the easiest way to get started with Dagster, this deployment type works best with jobs that aren't computationally expensive. Refer to the [Serverless guide](/dagster-cloud/deployment/serverless) for more info.
```

Comment:
Curious why we capitalize `Serverless` but not `hybrid`, I guess the former is a specific product while the latter is a descriptor? It does feel a little inconsistent though.

Reply:
Just an oversight - it'll be addressed in the re-write